### PR TITLE
allow multiple tinyMce open without submit

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: MediaAdminBundle/Resources/public/coffee/bindExtendView.coffee [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
-          target: target
-          input: editor.id
-          url: $('#' + target).data('url')
-        view.delegateEvents()
-        return
-      return
   return
+
+#ACTIVATE TINYMCE
+callback_tinymce_init = null
```
Comment (nicolasThal) [inline MediaAdminBundle/Resources/public/coffee/bindExtendView.coffee]: Does this part needs to be in this file? or can it be split into another file?

Comment (juchi) [inline MediaAdminBundle/Resources/public/coffee/bindExtendView.coffee]: The naming of the variable is not consistent with the rest of the project (CamelCase ?)
The variable should be assigned with a function, not null. This function could throw an exception or a console.error to inform that it has not been initiliazed.

Comment (itkg-nanne) [inline MediaAdminBundle/Resources/public/coffee/bindExtendView.coffee]: Could be split, but launched by the ready bind. That's why i choose to put this here. As you wish...

**Comment by juchi** [comment]
As a side note, I'd advise against global variables and functions.

File: MediaAdminBundle/Resources/public/coffee/bindExtendView.coffee (hunk 1 above)
Comment (itkg-nanne) [inline MediaAdminBundle/Resources/public/coffee/bindExtendView.coffee]: @juchi , callback_tinymce_init comes from tinyMce. We can't change it. Automatically called when tinyMce loaded.

**Comment by itkg-nanne** [comment]
@juchi : https://github.com/stfalcon/TinymceBundle#init-event

**Comment by juchi** [comment]
Ok, my bad.

**Pull request merged**